src/lib/LabelingMachine.ts

```diff
@@ -0,0 +1,32 @@
+// @ts-ignore Could not find a declaration file for module '@nimiq/iqons'.
+import { makeHash } from '@nimiq/iqons';
+// @ts-ignore Could not find a declaration file for module '@nimiq/iqons/dist/iqons-name.min.js'.
+import { name } from '@nimiq/iqons/dist/iqons-name.min.js';
+
+export default class LabelingMachine {
+    public static labelAddress(address: string): string {
+        return name(address);
+    }
+
+    public static labelAccount(firstAddress: string): string {
+        const colorNames = [
+            'Orange',
+            'Red',
+            'Yellow',
+            'Blue',
+            'Light-Blue',
+            'Purple',
+            'Green',
+            'Pink',
+            'Light-Green',
+            'Brown',
+        ];
+        const index = this.getBackgroundColorIndex(firstAddress);
+        return `${colorNames[index]} Account`;
+    }
+
+    private static getBackgroundColorIndex(address: string): number {
```

Comment:
I'd say, with Typescript, _private_ marked functions are actually private and throw a type error. The underscore is a convention for regular Javascript, where there is no distinction between public and private otherwise.

We actually have this not-consistent in the accounts code base. Some classes use underscores for private functions, others don't. 

I think it's fine without an underscore, since it's TS and it's marked private.

Reply:
This method is _gone_ now :rip: